extensions/mssql/src/utils.ts

```diff
@@ -95,24 +107,20 @@ export function getConfigTracingLevel(): string {
 	}
 }
 
-export function getDefaultLogDir(): string {
-	return path.join(process.env['ADS_LOGS'], '..', '..', 'mssql');
-}
-
-export function getDefaultLogFile(prefix: string, pid: number): string {
-	return path.join(getDefaultLogDir(), `${prefix}_${pid}.log`);
+export function getLogFileName(prefix: string, pid: number): string {
+	return `${prefix}_${pid}.log`;
 }
 
-export function getCommonLaunchArgsAndCleanupOldLogFiles(prefix: string, executablePath: string): string[] {
+export function getCommonLaunchArgsAndCleanupOldLogFiles(logPath: string, prefix: string, executablePath: string): string[] {
 	let launchArgs = [];
 	launchArgs.push('--log-file');
-	let logFile = getDefaultLogFile(prefix, process.pid);
+	let logFile = path.join(logPath, getLogFileName(prefix, process.pid));
 	launchArgs.push(logFile);
```

Comment:
Oh hey - we're not quoting this path which means it'll break for anyone with a space in the path. I'll make a separate issue for it but if you feel like fixing it now that'd be great =D